contracts/test/helpers/ProposalTest.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+import {Test, console2} from "forge-std/Test.sol";
+import {GovernorCountingSimpleUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorCountingSimpleUpgradeable.sol";
+import {CompoundGovernorTest} from "contracts/test/helpers/CompoundGovernorTest.sol";
+
+contract ProposalTest is CompoundGovernorTest {
```

Comment:
Why not bring all of this into `CompoundGovernorTest`? is it because we change to the new governor in `setUp`, which we don't want to do for all tests?

Reply:
My thinking was that not all tests would proposals, and those that did could import ProposalTest.sol, in and effort to keep CompountGovernorTest.sol smaller.